### DISCUSSION TITLE
410 Client Error

### DISCUSSION THREAD

**Discussion by thaiming97** [mampfes/hacs_waste_collection_schedule#3097] [Q&A] [answered]
I have a problem, I always get the following error message, when I select Landkreis Rhön Grabfeld

Die Quelle hat eine ungültige Antwort zurückgegeben: "410 Client Error: Gone for url: https://fs-api-rg.offizium.com/abfalltermine?stadt=****". Bitte überprüfe die Argumente und versuche es erneut.
(**** for privacy)

I think the website has changed, I found one where you can choose the location.

https://www.abfallinfo-rhoen-grabfeld.de/service/abfuhr-wecker

Or is another problem?

**Comment by 5ila5** [comment, marked as answer]
This was already reported (#3006) and fixed (#3015)

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
